Documentation/Meeting Notes/Group Meeting Minutes.md

```diff
@@ -68,3 +68,29 @@ Discussing the gitstats strategy
  - We've assigned the tasks for this week; we're primarily focusing on the database creation but Athena is setting up the mysql connection on the server and we've assigned you to create a front-end interface to show database results that are sent by the server
  - All the issues are updated on the github for more info
 
+
+
+## November 3rd
+###### 1:30-2:00pm: 
+Reviewing each other's work
+
+###### 2:00-2:45pm:
+Discussing the plan for next week
```

Comment:
When reading a meeting minutes file, the person reading it should have a decent understanding of the results of the meeting without having attended. For example, "we decided that each member should coordinate their meeting times for the week at the end of the meeting." Please take this feedback (and the feedback from last week) into account when writing these minutes. If you need help with understanding the expectations for the meeting minutes, please let us know.